packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -58,6 +55,11 @@ BuildArch: noarch
 
 Requires: subscription-manager
 Obsoletes: %{name}-fact-plugin < %{version}-%{release}
+Obsoletes: katello-agent
```

It's a best practice to include a version:
```suggestion
Obsoletes: katello-agent < %{version}-%{release}
```

This means that in the future it could be reintroduced. rpmlint should warn about this.